assets/app/index.html

```diff
@@ -65,7 +65,9 @@
               <li class="divider"></li>
               -->
               <li>
-                <a href="logout">Log out</a>
+                <!-- Stop event propagation to avoid a runtime error when the
+                     dropdown tries to close itself as we're navigating away. -->
+                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
```

Comment:
the error doesn't prevent you from going to logout does it?  wondering if we should just put in the ng-cloak stuff and leave the issue open as a P3 to get a fix into bootstrap -> patternfly